src/core/time.js

```diff
@@ -1,64 +1,60 @@
-Object.assign(pc, (function () {
+/**
+ * @private
+ * @class
+ * @name pc.Timer
+ * @description Create a new Timer instance.
+ * @classdesc A Timer counts milliseconds from when start() is called until when stop() is called.
+ */
+function Timer() {
+    this._isRunning = false;
+    this._a = 0;
+    this._b = 0;
+}
+
+Object.assign(Timer.prototype, {
     /**
      * @private
-     * @class
-     * @name pc.Timer
-     * @description Create a new Timer instance.
-     * @classdesc A Timer counts milliseconds from when start() is called until when stop() is called.
+     * @function
+     * @name pc.Timer#start
+     * @description Start the timer.
      */
-    var Timer = function Timer() {
-        this._isRunning = false;
-        this._a = 0;
-        this._b = 0;
-    };
-
-    Object.assign(Timer.prototype, {
-        /**
-         * @private
-         * @function
-         * @name pc.Timer#start
-         * @description Start the timer.
-         */
-        start: function () {
-            this._isRunning = true;
-            this._a = pc.now();
-        },
+    start: function () {
+        this._isRunning = true;
+        this._a = pc.now();
```

Comment:
`now()` only, same for `stop` method, and Timer needs to be exported to use `timer = new pc.Timer()` in devtools

Reply:
Well spotted! Fixed.